pkg/subctl/cmd/show_all.go

```diff
@@ -41,14 +41,23 @@ func showAll(cmd *cobra.Command, args []string) {
 		fmt.Println()
 		fmt.Printf("Showing information for cluster %q:\n", item.clusterName)
 
+		fmt.Println("Showing Network details")
 		showNetworkSingleCluster(item.config)
-		fmt.Println()
-		showEndpointsFromConfig(item.config)
-		fmt.Println()
-		showConnectionsFromConfig(item.config)
-		fmt.Println()
-		showGatewaysFromConfig(item.config)
-		fmt.Println()
-		showVersionsFromConfig(item.config)
+		fmt.Println("")
+
+		submariner := getSubmarinerResource(item.config)
+
+		if submariner == nil {
+			continue
```

Comment:
Can we print a line here before continuing explaining that this specific item.clusterName does not have submariner installed?

Reply:
getSubmarinerResource prints "Submariner is not installed" line.

Reply:
let's move that line here instead before continue, will make it more evident handling of the result/prints.